be/src/olap/rowset/segment_v2/segment_iterator.h

```diff
@@ -227,7 +281,10 @@ class SegmentIterator : public RowwiseIterator {
     std::unique_ptr<Schema> _seek_schema;
     // used to binary search the rowid for a given key
     // only used in `_get_row_ranges_by_keys`
-    std::unique_ptr<RowBlockV2> _seek_block;
+    vectorized::MutableColumns _seek_block;
+
+    //todo(wb) remove this field after Rowcursor is removed
+    vectorized::MutableColumns _short_key;
```

Comment:
_short_key -> _seek_key seems better？it's not **short** actually.

Reply:
In the existing Doris semantics,  short key means prefix index, you can refer ```ShortKeyIndexDecoder```.
Then seek block means it is used to seek data page in Segment,
Binary search can be realized by comparing seek block and short key.